b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/manageUserSection/otherComponents/addUserComponent.js

```diff
@@ -97,10 +105,28 @@ export default function AddUserComponent(prop) {
         errors = usernameValidate(values.username, errors);
         errors = passwordValidate(values.password, errors);
         errors = repasswordValidate(values.repassword, errors);
-        
+
         return errors;
     };
 
+    const inviteSelectOnChange = (value) => {
+        serInviteSelect(value);
+
+        switch (value) {
+            case InviteConst.INVITE:
+                setInviteShow(LOADING_DISPLAY_BLOCK);
+                setPasswordShow(LOADING_DISPLAY_NONE);
+
```

Comment:
Shall we remove the unwanted new lines?

Reply:
It is an eslint rule to add a space before `break`

### Rule
- Expected blank line before this statement.eslint[@typescript-eslint/padding-line-between-statements](https://typescript-eslint.io/rules/padding-line-between-statements)